src/Todo/Bundle/ListBundle/Entity/ListItem.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Todo\Bundle\ListBundle\Entity;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * List
+ *
+ * @ORM\Table(name="lists")
+ * @ORM\Entity(repositoryClass="Todo\Bundle\ListBundle\Repository\ListRepository")
+ */
+class ListItem
```

Comment:
`List` not `ListItem`

Reply:
php storm is considered List as a reserved word